lib/services/database.go

```diff
@@ -256,7 +340,19 @@ func engineToProtocol(engine string) string {
 	return ""
 }
 
-// parseRDSCustomEndpoint endpoint name from the provided RDS custom endpoint.
+// rdsEngineFamilyToProtocol converts RDS engine family to the database protocol.
+func rdsEngineFamilyToProtocol(engineFamily string) string {
```

Comment:
Isn't this the same as `engineToProtocol` function above?

Reply:
unfortunately no. `engineToProtocol` inputs are `"mysql"`, `"postgres"`, `"aurora"`, `"aurora-mysql"`, `"aurora-postgresql" `

engine families are: `"MYSQL" ` and `"POSTGRESQL"`